lte/gateway/python/magma/mobilityd/main.py

```diff
@@ -160,31 +160,32 @@ def main():
     # Load IPAddressManager
     ip_address_man = IPAddressManager(ipv4_allocator, ipv6_allocator, store)
 
-    # Add all servicers to the server
-    mobility_service_servicer = MobilityServiceRpcServicer(
-        ip_address_man, config.get('print_grpc_payload', False),
-    )
-    mobility_service_servicer.add_to_server(service.rpc_server)
-
     # Load IPv4 and IPv6 blocks from the configurable mconfig file
     # No dynamic reloading support for now, assume restart after updates
     ipv4_block = _get_ip_block(mconfig.ip_block, "IPv4")
     if ipv4_block is not None:
         logging.info('Adding IPv4 block')
         try:
-            mobility_service_servicer.add_ip_block(ipv4_block)
+            # Cleanup previously allocated IP blocks
+            allocated_ip_blocks = ip_address_man.list_added_ip_blocks()
+            ip_address_man.remove_ip_blocks(*allocated_ip_blocks, force=True)
+            ip_address_man.add_ip_block(ipv4_block)
```

Comment:
After this change, how do we maintain UE session IP address after service restart? this is expected in stateless mode.

Reply:
Added pre-check to only cleanup for different IP blocks